index.html

```diff
@@ -1,12 +1,14 @@
 <!DOCTYPE html>
 <html>
-
+<!--calls the css files into question-->
 <header>
     <title>Project Avatar</title>
-    <link rel="stylesheet" href="stylesheet.css">
+    <link href="/project-avatar/css/stylesheet.css" rel="stylesheet" type="text/css">
```

Comment:
Do you have /project-avatar/css/ folder with stylesheet.css file?

Reply:
No

Reply:
How am I even making folders? Everything's buttons instead of terminal controls